### PR TITLE
More from Timo's list.

### PR DESCRIPTION
**Pull request by bangerth**

### CODE REVIEW

File: publications-2018.bib [hunk 1]
```diff
@@ -1243,4 +1242,148 @@ @phdthesis{phdthesis-kauffman
 }
 
 
+@PhdThesis{Khattatov2018,
+  author =       {Eldar Khattatov},
+  title =        {Efficient discretization techniques and domain decomposition
+                  methods for poroelasticity},
+  school =       {University of Pittsburgh},
+  year =         2018},
```
Comment (tjhei) [inline publications-2018.bib]: missing brace?

**Comment by bangerth** [comment]
Fixed.

Is there a way I can test this locally? Could we add a `Makefile` or something that has a target `make check`?

**Comment by tjhei** [comment]
> Is there a way I can test this locally? Could we add a `Makefile` or something that has a target `make check`?

The Makefile in the root directory generates the publications.html but needs docker installed. Building latex is probably enough. I will add a target, but you can just do
```
               cd offline
               pdflatex -interaction=nonstopmode publication_list.tex
               biber publication_list
               pdflatex -interaction=nonstopmode publication_list.tex
```
https://github.com/dealii/publication-list/blob/master/Jenkinsfile#L69-L72

**Pull request merged**